common/utils/src/main/java/org/apache/spark/internal/Logger.java

```diff
@@ -193,4 +193,8 @@ static MessageThrowable of(String message, Throwable throwable) {
       return new MessageThrowable(message, throwable);
     }
   }
+
```

Comment:
The following code will use it:
<img width="509" alt="image" src="https://github.com/apache/spark/assets/15246973/095e1c1b-eea8-4082-9a68-58127a44442e">